src/server/shared/DataStores/DBCStructure.h

```diff
@@ -522,12 +529,19 @@ struct CreatureSpellDataEntry
     //uint32    availability[MAX_CREATURE_SPELL_DATA_SLOT]; // 4-7      m_availability[4]
 };
 
+enum CreatureTypeEntryFlags
+{
+    CREATURE_TYPE_ENTRY_FLAGS_IGNORED_TAB_TARGETING = 0x01 // Means do not include in tab targeting.
+};
+
 struct CreatureTypeEntry
 {
     uint32    ID;                                           // 0        m_ID
     //char*   Name[16];                                     // 1-16     name
                                                             // 17       string flags
-    //uint32    no_expirience;                              // 18 no exp? critters, non-combat pets, gas cloud.
+    uint32    Flags;                                        // 18 Critters, non-combat pets, gas cloud. (called Flags in struct)
```

Comment:
There is no reason to load this field anymore

Reply:
True, but why is so much effort put into not loading parts of DBC? Their memory footprint seems to be nearly nothing.

Reply:
I have no idea